ui/tract-renderer/src/main/java/org/cru/godtools/tract/activity/TractActivity.java

```diff
@@ -163,12 +164,14 @@ public static Intent createIntent(@NonNull final Context context, @NonNull final
                                       @NonNull final Locale... languages) {
         final Bundle extras = new Bundle();
         populateExtras(extras, toolCode, languages);
-        return new Intent(context, TractActivity.class).putExtras(extras);
+        return new Intent(context, TractActivity.class).putExtras(extras); //.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
```

Comment:
@frett not sure if I should add the flag or not here since it has no available Activity when called.

Reply:
leave it off, I would say responsibility for adding that flag should come from the caller that is using the intent. This method is just meant to create a base intent to launch this activity.

Reply:
I believe this is used locally and by the shortcuts logic

Reply:
remove the commented `//.addFlags(...`